lib/OpenQA/Setup.pm

```diff
@@ -57,7 +57,8 @@ sub read_config {
             auto_clone_regex            => '^(cache failure|terminated prematurely): ',
         },
         rate_limits => {
-            search => 5,
+            overview => 5,
```

Comment:
So does this mean that users can not access any test overview page more than 5 times per minute? In that case the limit is in range of 10-100 times below what power users might do

Reply:
And considering this is more of a production tweak, maybe the default should be a very high value (possibly infinity)?

Reply:
I could think of 200

Reply:
200 a minute?

Reply:
Yes, 200/minute

Reply:
Then I must misunderstand something - if this is about the webui, 200 requests a minute don't sound like a user but like a web crawler

Reply:
I wouldn't outright deny web crawlers. We could point web crawlers to the API :) thinking of a "power user" having multiple pages open and using automatic page refresh. Not the best approach but considering that we don't provide the users good hints in the error message I opted for this high number. But actually I would favor a more helpful error message

Reply:
If someone opts to rely on 200 refreshes a minute, he's a moron trying to sabotage his colleagues.

Reply:
> But actually I would favor a more helpful error message

If that's your concern: I'm looking into that. But somehow I can't get the negociation for json and html to work. So I pushed my non-working attempt now anyway, and maybe someone can spot where I'm failing.

Reply:
I like your error message but I consider 5 too low.

@Martchus you mentioned "possibly infinity", what do you propose?

Reply:
Since this is mainly a production tweak my suggestion is to disable rate limiting by default completely. That's what I meant with infinity here. Of course that is not a useful value if one actually wants to enable that feature.

Reply:
I think we can still propose a good default that we consider makes sense in most cases. So then let me suggest 50